### PR TITLE
Add a section for using Hadoop native library

### PR DESCRIPTION
**Pull request by bf8086**
Add a section in HDFS UFS page for using Hadoop native library.

### CODE REVIEW

**Comment by bf8086** [comment]
@calvinjia PTAL.

File: docs/en/ufs/HDFS.md [hunk 1]
```diff
@@ -246,3 +246,17 @@ alluxio.master.mount.table.root.option.alluxio.underfs.version=1.2
 Alluxio v{{site.ALLUXIO_RELEASED_VERSION}} supports the following versions of HDFS as a valid argument of mount option `alluxio.underfs.version`:
 
 - Apache Hadoop: 1.0, 1.2, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 3.0, 3.1
+
+### Use Hadoop Native Library
+
+Hadoop comes with native library that provides better performance and additional features to its Java implementation.
```
Comment (ZacBlanco) [inline docs/en/ufs/HDFS.md]: Suggestions:

- "hadoop provides a native library" or "comes with **a** native library"
- "features **compared** to its java implementation"

File: docs/en/ufs/HDFS.md [hunk 2]
```diff
@@ -246,3 +246,17 @@ alluxio.master.mount.table.root.option.alluxio.underfs.version=1.2
...
+For example, when native library is used, HDFS client can use native checksum function which is more efficient than default Java version.
```
Comment (ZacBlanco) [inline docs/en/ufs/HDFS.md]: Suggestions:
- "..when **the** native library..."
- "...**the** HDFS client can use **the/a** native checksum function which is more efficient than **the** default.."

File: docs/en/ufs/HDFS.md [hunk 3]
```diff
@@ -246,3 +246,17 @@ alluxio.master.mount.table.root.option.alluxio.underfs.version=1.2
...
+To use Hadoop native library with Alluxio HDFS under filesystem, first install the native library on Alluxio nodes by following the
```
Comment (ZacBlanco) [inline docs/en/ufs/HDFS.md]: - "To use **the** hadoop native library..."

File: docs/en/ufs/HDFS.md [hunk 4]
```diff
@@ -246,3 +246,17 @@ alluxio.master.mount.table.root.option.alluxio.underfs.version=1.2
...
+instructions on [this page](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/NativeLibraries.html).
+Once hadoop native library is installed on the machine, update Alluxio startup Java parameters in `conf/alluxio-env.sh`:
```
Comment (ZacBlanco) [inline docs/en/ufs/HDFS.md]: - "Once **the** hadoop native library..."

I would also me more explicit and tell the user to "add this line to the bottom of `conf/alluxio-env.sh`" or something like that

File: docs/en/ufs/HDFS.md (hunk 1 above)
Comment (bf8086) [reply docs/en/ufs/HDFS.md]: Updated.

File: docs/en/ufs/HDFS.md (hunk 2 above)
Comment (bf8086) [reply docs/en/ufs/HDFS.md]: Updated.

File: docs/en/ufs/HDFS.md (hunk 3 above)
Comment (bf8086) [reply docs/en/ufs/HDFS.md]: Updated.

File: docs/en/ufs/HDFS.md (hunk 4 above)
Comment (bf8086) [reply docs/en/ufs/HDFS.md]: Updated.

**Comment by bf8086** [comment]
Thanks @ZacBlanco! I updated the PR per your suggestions. PTAL.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**